var/spack/repos/builtin/packages/netcdf/package.py

```diff
@@ -44,10 +53,18 @@ class Netcdf(Package):
     # These variants control the number of dimensions (i.e. coordinates and
     # attributes) and variables (e.g. time, entity ID, number of coordinates)
     # that can be used in any particular NetCDF file.
-    variant('maxdims', default=1024,
-            description='Defines the maximum dimensions of NetCDF files.')
-    variant('maxvars', default=8192,
-            description='Defines the maximum variables of NetCDF files.')
+    variant(
+        'maxdims',
+        default=1024,
+        description='Defines the maximum dimensions of NetCDF files.',
+        values=is_integral
```

Comment:
what does `spack info netcdf` shows in terms of which values are allowed? I think it would be good to have a way for package developers to add a note what values are allowed in case it is checked via auxiliary function. Is it possible to fetch docstring from `is_integral` and show it in `spack info`? Alternatively a short optional description of allowed values would need to be provided within `variant()`, i.e. `description_values=`.